### PR TITLE
[5.11.0] - Fixed Configuring Identity Analytics Doc Issues

### PR DESCRIPTION
**Pull request by Sachin-Mamoru**
## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/learn/configuring-identity-analytics.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Using the WSO2 Identity Server Analytics distribution, you can view and analyze
 
 A taxi company called "Pickup" has launched a new application to be used by their customers. The Pickup developers wish to measure the performance of the authentication mechanism used for users to log into the application so that they can improve the user login experience. 
 
-To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application. 
+To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can setup WSO2 IS Analytics to view login attempts to the Pickup application.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-identity-analytics.md:9]:
```suggestion
To do this, the developers need to view authentication statistics about the login attempts to the application. This tutorial demonstrates how the Pickup developers can set up WSO2 IS Analytics to view login attempts to the Pickup application.
```

File: en/docs/learn/configuring-identity-analytics.md [hunk 2]
```diff
@@ -69,11 +102,11 @@ If you do not need to change the default values, proceed to start the servers.
 
 Let's create some basic authentication statistics. To do this, log in to the WSO2 IS dashboard. This login attempt will be published to WSO2 IS Analytics and you will be able to view the login attempt using the WSO2 IS Analytics dashboard. 
 
-1. Log in to the [WSO2 Identity Server User Portal](https://localhost:9443/user-portal/) using admin/admin credentials.
```
Comment (Sachin-Mamoru) [inline en/docs/learn/configuring-identity-analytics.md:72]: keep user portal

File: en/docs/learn/configuring-identity-analytics.md [hunk 3]
```diff
@@ -15,18 +15,51 @@ To do this, the developers need to view authentication statistics about the logi
...
+        `             <IS_HOME>/repository/resources/security            `
+        directory and run the following command.
+
+        !!! info 
+			Replace the `              <ISANALTICS_HOME>             ` placeholder
+			in the command with the filepath location of your
+			`              <ISANALYTICS_HOME>             ` folder.
+
+        ``` java
+        keytool -import -alias certalias -file <ISANALYTICS_HOME>/resources/security/sp.cer -keystore client-truststore.jks -storepass wso2carbon
```
Comment (Sachin-Mamoru) [inline en/docs/learn/configuring-identity-analytics.md:60]: change to .pem certificate

File: en/docs/learn/configuring-identity-analytics.md [hunk 4]
```diff
@@ -69,11 +102,11 @@ If you do not need to change the default values, proceed to start the servers.
 
 Let's create some basic authentication statistics. To do this, log in to the WSO2 IS dashboard. This login attempt will be published to WSO2 IS Analytics and you will be able to view the login attempt using the WSO2 IS Analytics dashboard. 
 
-1. Log in to the [WSO2 Identity Server User Portal](https://localhost:9443/user-portal/) using admin/admin credentials. 
+1. Log in to the [WSO2 Identity Server My Account](https://localhost:9443/myaccount/) using admin/admin credentials.
```
Comment (IsankaSR) [inline en/docs/learn/configuring-identity-analytics.md:105]: In 5.11, we use `User portal ` for `My Account`. Dont change it.

File: en/docs/learn/configuring-identity-analytics.md [hunk 5]
```diff
@@ -15,18 +15,51 @@ To do this, the developers need to view authentication statistics about the logi
...
+    `           client­-truststore.jks          ` of the WSO2 IS. This example uses the default keystores and
+    certificates.
+
+    1.  Navigate to the
+        `             <ISANALYTICS_HOME>/resources/security            `
+        directory on a new terminal window and run the following
+        command. The default keystore password is `wso2carbon`.
+
+        ``` java
+        keytool -export -alias wso2carbon -keystore wso2carbon.jks -file sp.cer
```
Comment (IsankaSR) [inline en/docs/learn/configuring-identity-analytics.md:47]: Use the .pem as in this doc https://is.docs.wso2.com/en/latest/learn/using-wso2-is-analytics-for-adaptive-authentication/

**Review by IsankaSR — APPROVED** [review]

**Pull request merged**